srcpkgs/sequoia-git/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'sequoia-git'
+pkgname=sequoia-git
+version=0.1.0
+revision=1
+build_style=cargo
+build_helper=qemu
+_deps="nettle-devel openssl-devel sqlite-devel"
```

Comment:
probably missing libgit2-devel (+ git2 crate patch) and bzip2-devel

Reply:
Thanks for catching these! Will rebase onto #45976, because it complains about libgit2 being too old. Is that the patch you were talking about?

Reply:
That one I'm not sure when I'll finish, I meant a downgrade patch like in e.g. https://github.com/void-linux/void-packages/tree/e53d2e9b28a449ad4e52bc0d4520454293873e4a/srcpkgs/cocogitto (patch that bumps down git2 version in Cargo.toml + a `post_patch()` with `cargo update --package git2@0.18.0 --precise 0.17.2`)

Reply:
I'm in no hurry when it comes to this package being merged. I've locally installed a version with a vendored libgit2 for now.